SQL/New_patches/2023_07_19_Add_Media_Digest.sql

```diff
@@ -0,0 +1,28 @@
+INSERT INTO notification_modules
```

Comment:
Should this PR be modifying the default schema or raisinbread too?

@ridz1208 ?